.github/ISSUE_TEMPLATE/epic.md

```diff
@@ -3,6 +3,7 @@ name: Epic
 about: Create an epic
 title: ''
 type: Epic
+projects: ["Jahia/18"]
```

Comment:
Interesting, maybe we could also add this for the bugs and others issues (Bugs triage project - 13) ?